Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,261 @@
+    - My conversation skills have surely improved and I feel more confident while joining online meetings. Also, I am more mindful of the deadlines and enjoyed solving challenges faced in this project.
+    - How to write a good proposal.
+    - How to work with a large community like this which is spread over different time zones.
+    - How to reach out to people, work with them, and solve each other's problems.
+    - I have become a better developer, not only in term of technical skills but in thinking of actual application of the product build and the edge case scenarios the user might face. Another major learning has been to speak up and ask for help or anything required which I previously was not too confident of.
+- Why students want to stay
+    - Community
+      - I had a great experience with Oppia and would like to continue contributing to the project. I'd like to help newer contributors too, (for example by reviewing their code when I get the chance).
+      - The organisation is active and have strong community bond.
+      - It is really an awesome experience working with some amazing folks from all around the world at Oppia.
+      - The kind of support the complete community provides is extra-ordinary.
+    - Giving back
+      - "Everybody wants to do some good work but lets be honest, it is maybe the lazyness or the time issue that you are not able to do the type of work you always wanted to do, what can be a good way to give back to the community than to contributing here.
+      - It makes me very happy that I'm a part of an organization which provides free education and I think the education is the biggest blessing we can give to one to make them stand on their feet.
+      - I would love to be part of this org by knowing that maybe not much but yes I'm trying to make an impact and my contribution in the educational field. I really want to do this because where I come from there is not much of education.
+      - Also the main reason to stay connected is the purpose the community serves. Providing education to those who do not have access to it helps me give back to the society.
+    - Growth / learning:
+      - Working with Oppia has really helped me grow as a developer and I would really like to stick around to gain even more experience of real world software development.
+      - Yes, the kind of work that Oppia does is really inspiring and there are a lot of opportunities to improve your skills be it be technical skills or leadership skills and most of all the people at Oppia are really fun to work with :)
+      - I like working in Oppia since it not only helps me improve my coding skills but also helps me grow as an individual.
+
+
+
+## GSoC Proposal Template
+
+When submitting a proposal, please use the provided [GSoC proposal template](https://docs.google.com/document/d/1yYefLkT7dJJa86MyrdWpbZtzeaWAKCi1eXZZDGUrasM). We will only consider proposals submitted using this template.
```

Comment:
Since we don't have the template ready for this year, should I keep a placeholder here?

Reply:
I'd leave out the link and include a placeholder stating that the template isn't ready yet. That way no one starts working based on the old template

Reply:
Here is the updated proposal template for 2023: https://docs.google.com/document/d/1ZZEJXCFgqPgPDuImKAW6EB9SMOS8cICPI2uCsVLww8w/edit# . Please review and leave any comments as you see fit -- but I think you can include it here.